scripts/bootloader/validation_data.py

```diff
@@ -79,25 +64,21 @@ def sign_and_append_validation_data(signature, input_file, public_key, offset, o
 
 def parse_args():
     parser = argparse.ArgumentParser(
-        description="Sign hex file and append metadata at specified offset.",
+        description="Append validation metadata at specified offset.",
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
     parser.add_argument("-i", "--input", required=True, type=argparse.FileType('r', encoding='UTF-8'),
-                        help="Hex file to sign.")
+                        help="Input hex file.")
     parser.add_argument("-o", "--offset", required=False, type=int,
```

Comment:
Might be out of scope, but -o should probably alias --output, not --offset :)

Reply:
Since we are the only one using this, I agree that we should do this breaking change